File: code/java/Code_3_04_1.java
import java.util.*;

class Code_3_04_1 {
	public static void main(String[] args) {
		// 輸入
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		int[] B = new int[N + 1];
		int[] R = new int[N + 1];
		for (int i = 1; i <= N; i++) {
			B[i] = sc.nextInt();
		}
		for (int i = 1; i <= N; i++) {
			R[i] = sc.nextInt();
		}
		
		// 答案的計算、輸出
		double blue = 0.0, red = 0.0;
		for (int i = 1; i <= N; i++) {
			blue += (double)B[i] / N;
			red += (double)R[i] / N;
		}
		System.out.format("%.12f\n", blue + red);
	}
}

File: code/c/Code_3_04_1.c
#include <stdio.h>

int N, B[100009], R[100009];

int main() {
	// 輸入
	scanf("%d", &N);
	int i;
	for (i = 1; i <= N; i++) {
		scanf("%d", &B[i]);
	}
	for (i = 1; i <= N; i++) {
		scanf("%d", &R[i]);
	}

	// 答案的計算 → 答案的輸出
	double blue = 0.0, red = 0.0;
	for (i = 1; i <= N; i++) {
		blue += 1.0 * B[i] / N;
		red += 1.0 * R[i] / N;
	}
	printf("%.12lf\n", blue + red);
	return 0;
}
